ngraph/core/src/op/experimental_detectron_prior_grid_generator.cpp

```diff
@@ -95,8 +95,9 @@ void op::v6::ExperimentalDetectronPriorGridGenerator::validate()
 
     const auto num_batches_featmap = featmap_shape[0];
     const auto num_batches_im_data = im_data_shape[0];
+    const auto batches_intersection = num_batches_featmap & num_batches_im_data;
```

Comment:
Why do you use logical and operation?

Reply:
It is not logical and operation, it is an intersection of intervals that represent dimensions `featmap_shape[0]` and `im_data_shape[0]`.

Reply:
Sure It a typo, of course I asked about binary and operation.